mne_bids/write.py

```diff
@@ -1956,6 +1956,11 @@ def write_raw_bids(
 
     # For the remaining files, we can use BIDSPath to alter.
     readme_fname = op.join(bids_path.root, "README")
+    allowed_readme_extensions = ("", ".md", ".txt", ".rst")
+    for cur_extension in allowed_readme_extensions:
+        if Path(bids_path.root, f"README{cur_extension}").exists():
+            readme_fname = str(Path(bids_path.root, f"README{cur_extension}"))
+            break
```

Comment:
```suggestion
    readme_suffixes = ("", ".md", ".rst", ".txt")
    found_readmes = sorted(
        filter(lambda x: x.suffix in readme_suffixes, bids_path.root.glob("README*"))
    )
    readme_fname = str((found_readmes or [bids_path.root / "README"])[0])
```

This suggestion seems a bit cleaner to me, but may be just stylistic preference; feel free to disregard @sappelhoff if you prefer @thht's approach. Not sure if the enclosing cast to `str` is necessary but that's what `op.join` would have returned. Behavior is slightly different if (for some reason) multiple READMEs are present; PR as written prefers `README` > `README.md` > `README.txt` > `README.rst`, whereas my suggestion prefers `.rst` over `.txt` if both are present. I doubt this matters; IIUC multiple READMEs present is considered a violation of the standard and should be very rare --- nonetheless do we want to detect this and error if it happens?

NB: Py3.12 adds a `case_sensitive=False` option to `Path.glob` that might be worth adding once 3.12 is minimum version.

Reply:
The cast to `str` is actually not necessary. I removed it.

Regarding possible multiple READMEs: I'm fine with both solutions. And I'd also leave it up to @sappelhoff if we should warn or throw if multiple READMEs are found....

Reply:
So, I adapted the approach by @drammock and added the check and exception raising when multiple readmes are found. I also added a test for this.